app/controllers/posts_controller.rb

```diff
@@ -72,7 +72,7 @@ def mark
   def hidden
     @hidden_boardviews = BoardView.where(user_id: current_user.id).where(ignored: true).includes(:board)
     hidden_post_ids = PostView.where(user_id: current_user.id).where(ignored: true).select(:post_id).distinct.pluck(:post_id)
-    @hidden_posts = posts_from_relation(Post.where(id: hidden_post_ids))
+    @hidden_posts = posts_from_relation(Post.where(id: hidden_post_ids).ordered)
     @page_title = 'Hidden Posts & Boards'
```

Comment:
Oh, huh, this should be "Continuities", not "Boards". Unrelated to this PR; I'll make an issue for it and to check over the codebase for more instances of this.